aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -50,19 +48,13 @@ The <xref:Microsoft.Extensions.Hosting.IHostedService> interface defines two met
 
 ### `StartAsync`
 
-`StartAsync` contains the logic to start the background task. `StartAsync` is called *before*:
+[StartAsync(CancellationToken)](xref:Microsoft.Extensions.Hosting.IHostedService.StartAsync%2A) contains the logic to start the background task. `StartAsync` is called *before*:
 
 * The app's request processing pipeline is configured.
 * The server is started and [IApplicationLifetime.ApplicationStarted](xref:Microsoft.AspNetCore.Hosting.IApplicationLifetime.ApplicationStarted%2A) is triggered.
 
-The default behavior can be changed so that the hosted service's `StartAsync` runs after the app's pipeline has been configured and `ApplicationStarted` is called. To change the default behavior, add the hosted service (`VideosWatcher` in the following example) after calling `CreateBuilder`:
+`StartAsync` should be limited to short running tasks because hosted services are run sequentially, and no further services are started until `StartAsync` runs to completion.
```

Comment:
I've pulled this up from `BackgroundService`, as it's specific to `StartyAsync`. `BackgroundService` implements `StartAsync`, but implementors of `BackgroundService` wouldn't typically override `StartAsync`.